### PR TITLE
HUB-720 Fixes build issues with verify-local-startup

### PR DESCRIPTION
**Pull request by j-maynard**
This fixes the build issues with the stub-event-sink in verify-local-startup.  The stub-event-sink.yml is empty by design.  The way verify-local-startup works is it makes assumptions on how the individual hub components are built and the presence of their configuration files.  These changes bring stub-event-sink inline with the other hub components.

### CODE REVIEW

**Review by CharlesIC — COMMENTED** [review]
I believe this new set of config should live in the `verify-local-startup` repo

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ARG conf_dir=configuration
+ARG hub_app
```
Comment (philandstuff) [inline Dockerfile:2]: I don't think we want `ARG` lines before `FROM`.  They are only visible in the `FROM` line itself, and the `FROM` doesn't reference them.

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -26,6 +28,7 @@ USER root
 ENV GRADLE_USER_HOME /usr/gradle/.gradle
 
 ARG hub_app
+ARG conf_dir
```
Comment (philandstuff) [inline Dockerfile:31]: This ARG is not referenced anywhere, so we should remove it.

```suggestion
```

**Review by instantiator — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -58,10 +61,11 @@ RUN gradle --console=plain \
 
 FROM openjdk:11.0.6-jre
 ARG hub_app
+ARG conf_dir
```
Comment (philandstuff) [inline Dockerfile:64]:
```suggestion
ARG conf_dir=configuration
```
If we delete the lines at the top of the file, we need to reintroduce the default value here.

**Review by philandstuff — COMMENTED** [review]
Looks sensible.  A few cleanup comments.

**Review by philandstuff — APPROVED** [review]

**Review by instantiator — APPROVED** [review]

**Pull request merged**